river/Output.zig

```diff
@@ -42,6 +42,9 @@ const State = struct {
     /// A bit field of focused tags
     tags: u32,
 
+    /// Remembered version of tags (from last run)
+    last_tags: u32,
+
```

Comment:
This doesn't need to go in the state struct, as it doesn't need double-buffering. We can just put it in `Output` directly. Notice how `Output.current.last_tags` is never accessed in your code.

Reply:
I tried to do what you suggested. I must add I never coded in zig, its just so readable small edit like this is possible even for a novice. Works on my end still